MovieApp/MovieApp/App/Features/MovieSearch/MoviesViewController/ViewModels/MovieViewModel.swift

```diff
@@ -0,0 +1,15 @@
+import UIKit
+
+struct MovieViewModel {
+    
+    var imageUrl: String
+    let title: String
+    let description: String
+    
+    init(with model: MovieNetworkModel) {
+        imageUrl = "https://image.tmdb.org/t/p/original" + model.imageUrl
```

Comment:
MovieViewModel should not know the path for his urls. Please move this to presenter for now, later on we will do this on UC layer